config/stack_custom.yaml

```diff
@@ -276,3 +276,16 @@ atlas:
   build: YES
   version: release-stable
   repo: jcsda
+
+gsl:
+  build: YES
+  version: 2.6
```

Comment:
Is GSL a requirement for MET?
We are building GSL-lite.  Is that sufficient?  If not, no worries, but we should check.
Also, please move this up to where gsl-lite is so we can keep those two together.

Reply:
GSL is a requirement for MET.  GSL-lite is not sufficient.  I tried compiling with GSL-lite and it was not successful.

Reply:
I moved gsl up to be underneath gsl-lite.